client/ayon_harmony/plugins/publish/extract_render.py

```diff
@@ -118,23 +118,30 @@ def process(self, instance):
 
         # Generate representations.
         extension = collection.tail[1:]
+        files = list(collection)
+        is_image_sequence = len(files) > 1
         representation = {
             "name": extension,
             "ext": extension,
-            "files": list(collection),
+            "files": files if is_image_sequence else files[0],
             "stagingDir": path,
             "tags": ["review"],
             "fps": frame_rate
         }
-
-        thumbnail = {
-            "name": "thumbnail",
-            "ext": "png",
-            "files": os.path.basename(thumbnail_path),
-            "stagingDir": path,
-            "tags": ["thumbnail"]
-        }
-        instance.data["representations"] = [representation, thumbnail]
+        representations = [representation]
+
+        # Add thumbnail if it's an image sequence
+        if is_image_sequence:
```

Comment:
Isn't the thumbnail created all the time?

Reply:
I had an error saying a transfer is already running... I can share it if you want, but it is easy to test.